indexer/proc_stableToken.go

```diff
@@ -0,0 +1,97 @@
+package indexer
+
+import (
+	"context"
+
+	"github.com/celo-org/celo-blockchain/accounts/abi/bind"
+	"github.com/celo-org/celo-blockchain/log"
+	//"github.com/celo-org/eksportisto/utils"
+	//"github.com/celo-org/eksportisto/metrics"
+	"github.com/celo-org/kliento/contracts"
+	"github.com/celo-org/kliento/contracts/helpers"
+	"github.com/celo-org/kliento/registry"
+	"github.com/go-errors/errors"
+)
+
+type stableTokenProcessorFactory struct{}
+
+func (stableTokenProcessorFactory) InitProcessors(
+	ctx context.Context,
+	handler *blockHandler,
+) ([]Processor, error) {
+	stableToken, err := handler.registry.GetStableTokenContract(ctx, handler.blockNumber)
+	if err != nil {
+		return nil, errors.Wrap(err, 1)
+	}
+
+	return []Processor{
+		&stableTokenProcessor{
+			blockHandler: handler,
+			logger:       handler.logger.New("processor", "stableToken", "contract", "StableToken"),
+			stableToken: stableToken,
+		},
+	}, nil
+}
+
+type stableTokenProcessor struct {
+	*blockHandler
+	logger       log.Logger
+	stableToken *contracts.StableToken
+}
+
+func (proc *stableTokenProcessor) EventHandler() (registry.ContractID, EventHandler) {
+	return "", nil
+}
+
+func (proc *stableTokenProcessor) ShouldCollect() bool {
+	// TODO: How often should it collect the data? Per Block? Per Epoch?
```

Comment:
Please remove the TODO once you're sure that it'll be collecting once per block.